src/Swarm/Game/Values.hs

```diff
@@ -0,0 +1,35 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+-- |
+-- SPDX-License-Identifier: BSD-3-Clause
+module Swarm.Game.Values where
```

Comment:
It's not a big deal, but for the most part we have been following the convention where nouns in module names are singular (*e.g.* `Swarm.Game.Robot` instead of `Robots`), and in particular we have `Swarm.Language.Value`, so it seems odd to me to have this module name be plural.

Reply:
done